jans-cedarling/cedarling/src/authz/mod.rs

```diff
@@ -311,8 +308,171 @@ impl Authz {
                     context: request.context.clone(),
                     resource: resource_uid.to_string(),
                     entities: entities_json,
-                    person_authorize_info: user_authz_info,
-                    workload_authorize_info: workload_authz_info,
+                    authorize_info: vec![user_authz_info, workload_authz_info]
```

Comment:
No need to initialize a `Vec` here only to turn it into an iterator then collect it onto a new `Vec`.

```suggestion
                    authorize_info: [user_authz_info, workload_authz_info]
```

Reply:
resolved